src/python/crash_analysis/crash_comparer.py

```diff
@@ -53,9 +53,20 @@ def _similarity_ratio(string_1, string_2):
       1.0 * length_sum)
 
 
+def count_same_frames(first_frames, second_frames):
+  """Count number of frames which are the same (disregarding order)."""
```

Comment:
We should account for order, the matches should be top-down. I can never think of a case of out-of-order and it decreases false positives.
also, i am worried about cases when assertion stack gets grouped with functional crashes [like two frames match]. Can you please run this over current open chromium, oss-fuzz crashes and see if the new groups look correct. What also happens sometimes is we overgroup, like a1 and a2 get group, and a2 and a3 get group, and unrelated a1, a3 get grouped (a1, a2, a3 all in one group). this overgroup case has happened a lot in past and needs to be monitored. The reason being that only one testcase is filed in overgroup, so we try to lower false positive as much as possible.

Reply:
Re matching ordering, I can think of stack overflows here. At the same time I can't think of many cases where two frames match and it's a false positive, but I can add this to be safe.

Will see if I can generate new groups.